src/containers/layouts/social/profile7/profile7.component.tsx

```diff
@@ -0,0 +1,207 @@
+            photo={{ uri: profile.photo }}
+            name={`${profile.firstName} ${profile.lastName}`}
+            location={profile.location}
+          />
+          <View style={themedStyle.actionContainer}>
+            <Button
+              style={themedStyle.followButton}
+              icon={PersonAddIconFill}
+              onPress={this.onFollowPress}>
+              Follow
+            </Button>
+            <Button
+              style={themedStyle.messageButton}
+              status='white'
+              icon={MessageCircleIconFill}
+              onPress={this.onMessagePress}>
+              Message
+            </Button>
+          </View>
+          <ProfileSocials
+            style={themedStyle.profileSocials}
+            textStyle={themedStyle.socialsLabel}
+            followers={socials.followers}
+            following={socials.following}
+            posts={socials.posts}
+            onFollowersPress={this.onFollowersPress}
+            onFollowingPress={this.onFollowingPress}
+            onPostsPress={this.onPostsPress}
+          />
+        </ImageBackground>
+        <ScrollView>
+          <View
+            style={[themedStyle.profileSection, themedStyle.aboutSection]}>
+            <Text style={themedStyle.profileSectionLabel}>About</Text>
+            <Text style={[themedStyle.profileSectionContent, themedStyle.profileAboutLabel]}>{profile.about}</Text>
+          </View>
+          <View style={[themedStyle.profileSection, themedStyle.friendsSection]}>
+            <Text style={[themedStyle.profileSectionLabel, themedStyle.friendsSectionLabel]}>Friends</Text>
+            <ProfileFriendList
+              contentContainerStyle={[themedStyle.profileSectionContent, themedStyle.friendsList]}
+              data={profile.friends}
+              onItemPress={this.onFriendPress}
+            />
+          </View>
+          <View style={themedStyle.profileSection}>
+            <Text style={themedStyle.profileSectionLabel}>Shots</Text>
+            <ProfileActivityList2
+              contentContainerStyle={[themedStyle.profileSectionContent]}
+              data={activities.map(this.createActivitySource)}
+              onItemPress={this.onPhotoPress}
+            />
+          </View>
+        </ScrollView>
+      </View>
+    );
+  }
+}
+
+export const Profile7 = withStyles(Profile7Component, (theme: ThemeType) => ({
+  container: {
+    flex: 1,
+  },
+  profileInfoContainer: {
+    paddingHorizontal: 24,
+    paddingVertical: 24,
+    backgroundColor: theme['color-basic-100'],
+  },
+  profileSocials: {
+    justifyContent: 'space-evenly',
+    marginTop: 24,
+    paddingVertical: 16,
+  },
+  actionContainer: {
+    flexDirection: 'row',
+    marginTop: 32,
+  },
+  profileSection: {
+    marginTop: 32,
+  },
+  profileSectionContent: {
+    marginTop: 8,
+  },
+  socialsLabel: {
+    color: theme['color-white'],
+  },
+  followButton: {
+    flex: 1,
+    marginRight: 4,
+    textTransform: 'uppercase',
```

Comment:
I guess string value in button is static. Just write 'FOLLOW' and don't create extra fields in the objects.